### PR TITLE
Updating Governance and risk to Risk and compliance

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/11445

### CODE REVIEW

File: about/architecture.adoc [hunk 1]
```diff
@@ -50,13 +50,13 @@ A multicluster application uses the Kubernetes specification, but provides addit
...
 After you configure a {product-title-short} hub cluster and a managed cluster, you can view and create policies with the {product-title-short} policy framework. You can take advantage of the https://github.com/open-cluster-management/policy-collection/tree/master/community[`policy-collection` community] to see what policies community members created and contributed, as well as contribute your own policies for others to use. 
 
-For more information about Governance and risk, see the link:../risk_compliance/security_intro.adoc#security[Security] introduction. Additionally, learn about access requirements from the link:../authentication/rbac.adoc#role-based-access-control[Role-based access control] documentation.
+For more information about Risk and compliance, see the link:../risk_compliance/security_intro.adoc#security[Security] introduction. Additionally, learn about access requirements from the link:../authentication/rbac.adoc#role-based-access-control[Role-based access control] documentation.
```
Comment (cadawson) [inline about/architecture.adoc]: risk and compliance

File: risk_compliance/create_cert_pol.adoc [hunk 2]
```diff
@@ -63,7 +63,7 @@ As you create a certificate policy from the console, a YAML file is also created
 Complete the following steps to create a certificate policy from the console:
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_cert_pol.adoc:66]: Compliance

File: risk_compliance/create_cert_pol.adoc [hunk 3]
```diff
@@ -76,7 +76,7 @@ A certificate policy is created.
 You can view any certificate policy and its status from the console.
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_cert_pol.adoc:79]: Compliance

File: risk_compliance/create_cert_pol.adoc [hunk 4]
```diff
@@ -139,7 +139,7 @@ Complete the following steps to disable a certificate policy from the CLI or the
 
 * Disable a certificate policy from the console:
  .. Log in to your {product-title} console.
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_cert_pol.adoc:142]: Compliance

File: risk_compliance/create_cert_pol.adoc [hunk 5]
```diff
@@ -164,10 +164,10 @@ After your policy is deleted, it is removed from your target cluster or clusters
  .. Verify that your policy is removed by running the following command:
 +
 ----
-kubectl get policy <policy-name> -n <mcm namespace>
+kubectl get policy <policy-name> -n <namespace>
 ----
 * Delete a certificate policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_cert_pol.adoc:170]: Compliance

File: risk_compliance/create_config_pol.adoc [hunk 6]
```diff
@@ -83,7 +83,7 @@ As you create a configuration policy from the console, a YAML file is also creat
 Complete the following steps to create a configuration policy from the console:
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_config_pol.adoc:86]: Compliance

File: risk_compliance/create_config_pol.adoc [hunk 7]
```diff
@@ -101,7 +101,7 @@ Complete the following steps to create a configuration policy from the console:
 You can view any configuration policy and its status from the console.
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_config_pol.adoc:104]: Compliance

File: risk_compliance/create_config_pol.adoc [hunk 8]
```diff
@@ -118,7 +118,7 @@ Learn to update configuration policies by viewing the following section.
 Complete the following steps to disable your configuration policy:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_config_pol.adoc:121]: Compliance

File: risk_compliance/create_config_pol.adoc [hunk 9]
```diff
@@ -146,7 +146,7 @@ After your policy is deleted, it is removed from your target cluster or clusters
 kubectl get policy <policy-name> -n <namespace>
 ----
 * Delete a configuration policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_config_pol.adoc:149]: Compliance

File: risk_compliance/create_image_vuln.adoc [hunk 10]
```diff
@@ -63,7 +63,7 @@ As you create an image vulnerability policy from the console, a YAML file is als
 Complete the following steps to create the image vulnerability policy from the console:
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_image_vuln.adoc:66]: Compliance

File: risk_compliance/create_image_vuln.adoc [hunk 11]
```diff
@@ -75,7 +75,7 @@ An image vulnerability policy is created.
 [#viewing-image-vulnerability-violations-from-the-console]
 === Viewing image vulnerability violations from the console
 
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_image_vuln.adoc:78]: Compliance

File: risk_compliance/create_image_vuln.adoc [hunk 12]
```diff
@@ -100,7 +100,7 @@ Learn to update image vulnerability policies by viewing the following section.
 Complete the following steps to disable your image vulnerability policy:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_image_vuln.adoc:103]: Compliance

File: risk_compliance/create_image_vuln.adoc [hunk 13]
```diff
@@ -128,7 +128,7 @@ After your policy is deleted, it is removed from your target cluster or clusters
 kubectl get policy <imagevulnpolicy-name> -n <namespace>
 ----
 * Delete an image vulnerability policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_image_vuln.adoc:131]: Compliance

File: risk_compliance/create_memory_policy.adoc [hunk 14]
```diff
@@ -64,7 +64,7 @@ As you create a memory usage policy from the console, a YAML file is also create
 Complete the following steps to create the memory usage policy from the console:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_memory_policy.adoc:67]: Compliance

File: risk_compliance/create_memory_policy.adoc [hunk 15]
```diff
@@ -77,7 +77,7 @@ You can edit your values.
 You can view any memory usage policy and its status from the console.
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_memory_policy.adoc:80]: Compliance

File: risk_compliance/create_memory_policy.adoc [hunk 16]
```diff
@@ -95,7 +95,7 @@ Learn to update memory usage policies by viewing the following section.
 Complete the following steps to disable your memory usage policy:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_memory_policy.adoc:98]: Compliance

File: risk_compliance/create_memory_policy.adoc [hunk 17]
```diff
@@ -122,7 +122,7 @@ After your policy is deleted, it is removed from your target cluster or clusters
 kubectl get policy <memorypolicy-name> -n <namespace>
 ----
 * Delete a memory usage policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_memory_policy.adoc:125]: Compliance

File: risk_compliance/create_ns_policy.adoc [hunk 18]
```diff
@@ -62,7 +62,7 @@ As you create a namespace policy from the console, a YAML file is also created i
 Complete the following steps to create a namespace policy from the console:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_ns_policy.adoc:65]: Compliance

File: risk_compliance/create_ns_policy.adoc [hunk 19]
```diff
@@ -75,7 +75,7 @@ You can edit your values.
 You can view any namespace policy and its status from the console.
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_ns_policy.adoc:78]: Compliance

File: risk_compliance/create_ns_policy.adoc [hunk 20]
```diff
@@ -93,7 +93,7 @@ Learn to update namespace policies by viewing the following section.
 Complete the following steps to disable your namespace policy:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_ns_policy.adoc:96]: Compliance

File: risk_compliance/create_ns_policy.adoc [hunk 21]
```diff
@@ -121,7 +121,7 @@ After your policy is deleted, it is removed from your target cluster or clusters
  kubectl get policy <namespacepolicy-name> -n <namespace>
 ----
 * Delete a namespace policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_ns_policy.adoc:124]: Compliance

File: risk_compliance/create_policy.adoc [hunk 22]
```diff
@@ -140,7 +140,7 @@ kubectl describe securitypolicy <name> -n <namespace>
 
 As you create your new policy from the console, a YAML file is also created in the YAML editor.
 
-. From the navigation menu, click *Govern risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_policy.adoc:143]: Compliance

File: risk_compliance/create_policy.adoc [hunk 23]
```diff
@@ -226,7 +226,7 @@ A security policy is created from the console.
 You can view any security policy and its status from the console.
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_policy.adoc:229]: Compliance

File: risk_compliance/create_policy.adoc [hunk 24]
```diff
@@ -245,7 +245,7 @@ Learn to update security policies by viewing the following section.
 Your policy is enabled by default. You can disable your policy by completing the following steps:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_policy.adoc:248]: Compliance

File: risk_compliance/create_policy.adoc [hunk 25]
```diff
@@ -268,10 +268,10 @@ kubectl delete policy <securitypolicy-name> -n <open-cluster-management-namespac
 After your policy is deleted, it is removed from your target cluster or clusters. Verify that your policy is removed by running the following command: `kubectl get policy <securitypolicy-name> -n <open-cluster-management-namespace>`
 
 * Delete a security policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_policy.adoc:271]: Compliance

File: risk_compliance/create_policy.adoc [hunk 26]
```diff
@@ -268,10 +268,10 @@ kubectl delete policy <securitypolicy-name> -n <open-cluster-management-namespac
...
  .. Click the *Actions* icon for the policy you want to delete in the policy violation table.
  .. Click *Remove*.
  .. From the _Remove policy_ dialog box, click *Remove policy*
 
 To manage other policies, see xref:../risk_compliance/create_policy.adoc#managing-security-policies[Managing security policies] for more information.
-Refer to xref:../risk_compliance/grc_intro.adoc#governance-and-risk[Governance and risk] for more topics about policies.
+Refer to xref:../risk_compliance/grc_intro.adoc#governance-and-risk[Risk and compliance] for more topics about policies.
```
Comment (cadawson) [inline risk_compliance/create_policy.adoc:277]: Compliance

File: risk_compliance/create_rb_policy.adoc [hunk 27]
```diff
@@ -63,7 +63,7 @@ As you create a role binding policy from the console, a YAML file is also create
 Complete the following steps to create a role binding policy from the console:
 
 . Log in to your cluster from the console.
-. From the navigation menu, click *Governance and risk*.
+. From the navigation menu, click *Risk and compliance*.
```
Comment (cadawson) [inline risk_compliance/create_rb_policy.adoc:66]: Compliance

File: risk_compliance/create_rb_policy.adoc [hunk 28]
```diff
@@ -102,7 +102,7 @@ Learn to update role binding policies by viewing the following section.
 Complete the following steps to disable your role binding policy:
 
 . Log in to your {product-title} console.
-. From the navigation menu, click *Govern risk* to view a table list of your policies.
+. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_rb_policy.adoc:105]: Compliance

File: risk_compliance/create_rb_policy.adoc [hunk 29]
```diff
@@ -129,7 +129,7 @@ After your policy is deleted, it is removed from your target cluster or clusters
 kubectl get policy <rolebinding-policy-name> -n <namespace>
 ----
 * Delete a role binding policy from the console:
- .. From the navigation menu, click *Govern risk* to view a table list of your policies.
+ .. From the navigation menu, click *Risk and compliance* to view a table list of your policies.
```
Comment (cadawson) [inline risk_compliance/create_rb_policy.adoc:132]: Compliance

**Review by cadawson — APPROVED** [review]
Given our discussion about the UI nav, many of the changes here are likely to not be incorporated. There still might be some that are helpful. Looks good overall!

**Comment by dockerymick** [comment]
@cadawson thanks for the review and feedback. Based on our discussion in Slack, i am going to keep Risk and compliance. I also asked for dev to make this update. It isn't a big change, so if need be I will revert the capitalization change

**Pull request merged**